packages/documentation/pages/usage/components/button.vue

```diff
@@ -409,12 +408,11 @@
 				<div>
 					<h4>Texts</h4>
 					<KtFieldText formKey="label" label="label" />
```

Comment:
This should now have the `isRequired` prop.

There should be a few more cases like this, when I look for `<KtField` in the code base I see 211 hits, searching for `isOptional` yields 195 hits, that implies about 16 cases where the `isRequired` props needs to be added (there are probably some false flags). I think that makes the change more none-trivial and could introduce unintended bugs. Maybe we should not risk it?

Reply:
Or we need a somewhat sophisticated migration tool that handles inserting `isRequired` where needed, but that sounds non-trivial. Only Idea I have is to create a custom eslint rule that at least marks all usages of a KtField* component that neither has `isRequired` nor `isOptional` prop

Reply:
I say let's do this at a later time.

Reply:
Agreed